pkgs/by-name/jt/jtdx/package.nix

```diff
@@ -0,0 +1,11 @@
+{ wsjtx, fetchgit, qt5 }:
+wsjtx.overrideAttrs (old: {
```

Comment:
I can see the logic behind leveraging the existing `wsjtx` drv since `jtdx` is a derivative work, but it would be a good idea to include a specific `jtdx` `meta` attribute. At which point are we really gaining anything by using `overrideAttrs`? Heres the `wsjtx` drv for context: https://github.com/NixOS/nixpkgs/blob/2cf2f3a306f5ea1f0e184b4317c4d50679ec49b3/pkgs/applications/radio/wsjtx/default.nix

Reply:
I'm using this pattern because it's a derivative work, and it's how the Godot derivations used to work, and I think it's a good pattern. It's also used for ioq3-scion. I've added the missing metadata though!